### ISSUE TITLE
Topology tests - nodes should not be able to adjust to nodes that have not started

### ISSUE THREAD

**Issue by rslawson** [bittide/bittide-hardware#656]
Going by the graphs from the topology tests (both software and hardware) appear to allow nodes to adjust to the frequency of nodes that have not finished their delay period. Of note is the hourglass topology - as @kleinreact pointed out, it appears that nodes with no startup delay (starting at -100ppm relative frequency) are adjusting to the frequency of the nodes that have a 1.6s startup delay:
![image](https://github.com/user-attachments/assets/f6a730c5-ee67-4a2d-8d7d-1634d47e404e)

Probably worth fixing.

**Comment by rslawson** [comment]
This is now fixed in the SwCc branch for the software topologies test (#654). I'll add the fix into the HwCc test separately in another PR.

**Comment by rslawson** [comment]
Scratch that, actually. I forgot that once SwCc is merged, the HwCc tests will be removed, so that's unnecessary.
